invertedai/helpers/waypoints.py

```diff
@@ -124,74 +319,74 @@ def get_lanelet(id):
     waypoints = [Point(x=x, y=y) for x, y in zip(new_x, new_y)]
 
     return waypoints
-    
 
 def generate_lane_ids_from_lanelet_map(
     start_state: AgentState, 
     lanelet_map: lanelet2.core.LaneletMapLayers, 
-    target_distance: Optional[float] = None ,
-    waypoint: Optional[Point] = None
+    min_distance: Optional[float] = None, 
```

Comment:
If `min_distance` is optional, then we should not assume a fixed value inside the method.